mcodingbot/plugins/python.py

```diff
@@ -29,9 +30,10 @@ def get_dismiss_button(id: hikari.Snowflake) -> hikari.api.ActionRowBuilder:
     return action_row
 
 
-def get_pep_link(pep_number: int, *, hide_embed: bool) -> str:
-    url = f"https://peps.python.org/pep-{pep_number:04}/"
-    return f"<{url}>" if hide_embed else url
+@plugin.include
+@tasks.cronjob("@daily")
```

Comment:
Might be a good idea to set this to a specific time, like when the server is least active? I don't know how much of a load this task takes on the bot, but running it at midnight makes the most sense to me.

If it's too much work to set a specific time dw about it.

Reply:
Midnight? UTC?

Reply:
I dont think theres server activity data. Also time have to be adjusted for the bots timezone so thats already kinda sus.

Reply:
Seems the server is basically inactivate at midnight est

Reply:
*based on messages in #general

Reply:
> Also time have to be adjusted for the bots timezone so thats already kinda sus.

yeah dw about it I guess, it'll be fine as it is